src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -19,6 +19,11 @@ public class LanguageParserFixture : CoreTest
         [TestCase("2 Broke Girls - S01E01 - Pilot.eng.sub")]
         [TestCase("2 Broke Girls - S01E01 - Pilot.English.sub")]
         [TestCase("2 Broke Girls - S01E01 - Pilot.english.sub")]
+        [TestCase("The.Spanish.Princess.S02E08.Peace.1080p.AMZN.WEBRip.DDP5.1.x264-NTb")]
+        [TestCase("The Spanish Princess S02E02 Flodden 720p AMZN WEB-DL DDP5 1 H 264-NTb")]
+        [TestCase("The.Spanish.Princess.S02E07.1080p.WEB.H264-GGWP")]
+        [TestCase("The.Spanish.Princess.S02E02.1080p.WEB.H264-CAKES")]
+        [TestCase("The.Spanish.Princess.S02E06.Field.of.Cloth.of.Gold.1080p.AMZN.WEBRip.DDP5.1.x264-NTb")]
```

Comment:
These cases can be trimmed down to just:

```
[TestCase("The Spanish Princess S02E02 Flodden 720p AMZN WEB-DL DDP5 1 H 264-NTb")]
[TestCase("The.Spanish.Princess.S02E02.1080p.WEB.H264-CAKES")]
```

maybe:
```
[TestCase("The.Spanish.Princess.S02E06.Field.of.Cloth.of.Gold.1080p.AMZN.WEBRip.DDP5.1.x264-NTb")]
```
as well, but everything else is is covered by these 2-3.